setup.py

```diff
@@ -29,7 +29,7 @@
 
 
 setup(name='dymos',
-    version='0.16.1-dev',
+    version='version='0.16.2-dev',-dev',
```

Comment:
Did something get mangled here?

Reply:
Hmm good catch, will check it after I pick up my kids